tests/integration/account/test_account.py

```diff
@@ -64,6 +71,7 @@ def get_group(display_name: str) -> Group:
 
     group = get_group(group_display_name)
     assert group
+    assert len(group.members) == 2  # 2 members: user and service principal
```

Comment:
Nit, if possible I sometimes use something like this instead:
```suggestion
    expected_members = {member_a, member_b}
    assert set(group.members) == {member_a, member_b} and len(group.members) == len(expected_members)
```
(Verifies both length and content.)